docs/docs/99-api/examples/automation_demo/more_specific_demos/demo_C.js

```diff
@@ -10,13 +10,7 @@
  *      - ~/examples/automation_demo/more_specific_demos/draw_automated.svg
  */
 
-const {
```

Comment:
This change is reasonable but not related to this PR, so please revert it and make a separate PR.